repos/system_upgrade/el8toel9/actors/opensshsubsystemsftp/libraries/opensshsubsystemsftp.py

```diff
@@ -0,0 +1,47 @@
+from leapp import reporting
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api
+
+
+def process(openssh_messages):
+    config = next(openssh_messages, None)
+    if list(openssh_messages):
+        api.current_logger().warning('Unexpectedly received more than one OpenSshConfig message.')
+    if not config:
+        raise StopActorExecutionError(
+            'Could not check openssh configuration', details={'details': 'No OpenSshConfig facts found.'}
+        )
+
+    # not modified configuration will get updated by RPM automatically
+    if not config.modified:
+        return
+
+    if not config.subsystem_sftp:
+        resources = [
+            reporting.RelatedResource('package', 'openssh-server'),
+            reporting.RelatedResource('file', '/etc/ssh/sshd_config'),
+            reporting.ExternalLink(
+                title="SCP support in RHEL",
+                url="https://access.redhat.com/articles/5284081",
+            ),
+            # TODO provide a link to documentation or blog post
```

Comment:
@Jakuje is it the TODO resolved by the link above?

Reply:
no, the above is generic information about SCP. I wanted to add something more specific regarding to the deprecation. I will try to find the correct one.

Reply:
@pirat89 This should be the one I had in my mind when writing the todo: https://www.redhat.com/en/blog/openssh-scp-deprecation-rhel-9-what-you-need-know
Should I add a separate PR or can you update it somehow on your own?

Reply:
@Jakuje  create please the separate PR. thanks

Reply:
Filled as #893